.github/workflows/integrationTest.yml

```diff
@@ -0,0 +1,200 @@
+name: Deploy app to ubuntu server
+
```

Comment:
see `build.yml` for event structure.  Should be push branch master and pull_request with specific types.

Reply:
We can't run this flow on pr since this is a public repo. Github secrets can't be shared. Each person will need to provide their own secrets to have this work on pushes.

Reply:
Made the change